pkg/snowflake/procedure.go

```diff
@@ -230,7 +230,7 @@ type procedure struct {
 	Name         sql.NullString `db:"name"`
 	SchemaName   sql.NullString `db:"schema_name"`
 	Text         sql.NullString `db:"text"`
-	DatabaseName sql.NullString `db:"database_name"`
+	DatabaseName sql.NullString `db:"catalog_name"`
```

Comment:
This looks like a mistake?

Reply:
For some reason, Snowflake returns the database name in a column called `catalog_name` when calling the SHOW PROCEDURES query.
https://docs.snowflake.com/en/sql-reference/sql/show-procedures.html#output

Reply:
well that's weird and not uniform...